tracim/views/core_api/schemas.py

```diff
@@ -9,23 +9,43 @@
 from tracim.models.contents import GlobalStatus
 from tracim.models.contents import open_status
 from tracim.models.context_models import ContentCreation
+from tracim.models.context_models import SetContentStatus
+from tracim.models.context_models import CommentCreation
+from tracim.models.context_models import CommentPath
 from tracim.models.context_models import MoveParams
 from tracim.models.context_models import WorkspaceAndContentPath
 from tracim.models.context_models import ContentFilter
 from tracim.models.context_models import LoginCredentials
+from tracim.models.context_models import HTMLDocumentUpdate
+from tracim.models.context_models import ThreadUpdate
 from tracim.models.data import UserRoleInWorkspace
 
 
-class UserSchema(marshmallow.Schema):
-
+class UserDigestSchema(marshmallow.Schema):
+    """
+    Simple user schema
+    """
     user_id = marshmallow.fields.Int(dump_only=True, example=3)
-    email = marshmallow.fields.Email(
-        required=True,
-        example='suri.cate@algoo.fr'
+    avatar_url = marshmallow.fields.Url(
+        allow_none=True,
+        example="/api/v2/assets/avatars/suri-cate.jpg",
+        description="avatar_url is the url to the image file. "
+                    "If no avatar, then set it to null "
+                    "(and frontend will interpret this with a default avatar)",
     )
     public_name = marshmallow.fields.String(
         example='Suri Cate',
     )
+
+
+class UserSchema(UserDigestSchema):
+    """
+    Complete user schema
+    """
+    email = marshmallow.fields.Email(
+        required=True,
+        example='suri.cate@algoo.fr'
+    )
```

Comment:
```
created = marshmallow.fields.DateTime(
    format='%Y-%m-%dT%H:%M:%SZ',
    [...]
```

The datetime format MUST be hard written one time only in **the entire tracim source code**

you must use it like `from tracim.utils import DATETIME_FORMAT`

Reply:
i can use this value for tests ?